src/common/group_data.h

```diff
@@ -71,23 +71,23 @@ struct ParallelGroupBuilder {
     // set rptr to correct size
     for (std::size_t tid = 0; tid < thread_rptr_.size(); ++tid) {
       if (rptr_.size() <= thread_rptr_[tid].size()) {
-        rptr_.resize(thread_rptr_[tid].size() + 1);  // key + 1
+        rptr_.resize(thread_rptr_[tid].size() + 1, rptr_.back());  // key + 1
```

Comment:
Is it possible that `rptr_` has 0 size?  `back()` will segfault.

Reply:
Well spotted. Fixed and added tests.